airqo-platform/AirQo-api
# disable the Youtube video link from the Platform welcome

**_WHAT DOES THIS PR DO?_**
temporarily disable the Youtube video link from the Platform welcome emails.
Mainly because the content on that channel is not actively maintained.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [PLAT-1204]

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/device-registry
npm install
npm run stage-mac
```
check README for OS specific run instructions

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [ create User](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-8cfe27b0-0d50-4c72-b809-b728172d9716)




[PLAT-1204]: https://airqoteam.atlassian.net/browse/PLAT-1204?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
